docs/en/stack/ml/nlp/ml-nlp-elser.asciidoc

```diff
@@ -385,6 +385,35 @@ The following sections provide information about how ELSER performs on different
 hardwares and compares the model performance to {es} BM25 and other strong 
 baselines such as Splade or OpenAI.
 
+[discrete]
+[[version-overview]]
+=== Version overview
+
+The ELSER model has two main versions: ELSER V2 and ELSER V1.
+ELSER V2 has a **platform specific** version that is designed to run on Linux 
+with an x86-64 CPU architecture and a **platform agnostic or portable** version 
+that can be run on any platform.
+
+[discrete]
+==== ELSER V2
+
+The biggest change in ELSER V2 is the introduction of the first platform 
```

Comment:
(optional) Perhaps it would be good to mention the performance improvements in V2 such as:
"The biggest change in ELSER V2 (other than the large improvements in performance) is the introduction of . . ."